.circleci/config.yml

```diff
@@ -1016,10 +1016,7 @@ jobs:
         type: string
         default: ""
     steps:
-      - slack-basic:
-          channel: C03STD3UXMG
-          event: always
-          slack_mentions_user: "@joaquin"
+      - slack-basic
```

Comment:
Caveat: this means we'll now get Starting and Completed/Failed announcements to #eng-deploys-announcments.

I think we should do this for a few days, but we should consider if we don't want these 2 messages for every commit to `master`.